src/frontend/json-per-line-parser.js

```diff
@@ -66,7 +66,7 @@ _.extend(JsonPerLineParser.prototype, {
 	},
 	
 	_onData: function (jsonObject) {
-		logger.log(this._options.logPrefix + 'Data:', jsonObject);
+		//logger.log(this._options.logPrefix + 'Data:', jsonObject);
```

Comment:
No, I did not. This piece is used for debugging when needed, but spams the logs when not needed – hence commented out. The comment will be stripped out from the production build.